model/event/host_event_finder.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 func getRecentStatusesForHost(hostId string, n int) (int, []string) {
+	or := eitherResourceTypeKeyIs(ResourceTypeHost)
+	or[TypeKey] = EventTaskFinished
```

Comment:
won't this cause queries to do collection scans?

Reply:
No, $or can use multiple indexes.